sql/internal/specutil/convert.go

```diff
@@ -279,6 +280,22 @@ func FromPrimaryKey(s *schema.Index) (*sqlspec.PrimaryKey, error) {
 	}, nil
 }
 
+// FromColumn converts a *schema.Column into a *sqlspec.Column using the ColumnTypeSpecFunc.
+func FromColumn(col *schema.Column, columnTypeSpec ColumnTypeSpecFunc) (*sqlspec.Column, error) {
```

Comment:
```suggestion
func FromColumn(col *schema.Column, convertType ColumnTypeSpecFunc) (*sqlspec.Column, error) {
```

The rest of the functions name it as `convert<T>`.

Reply:
actually we have <X>Spec for the ones going from schema to spec and convert<X> for other direction